src/main/java/com/endava/internship/collections/StudentList.java

```diff
@@ -1,140 +1,393 @@
 package com.endava.internship.collections;
 
+
 import java.util.Collection;
 import java.util.Iterator;
 import java.util.List;
+import java.util.Objects;
+import java.util.Arrays;
 import java.util.ListIterator;
 
+
 public class StudentList implements List<Student> {
+    private int capacity;
+    private int size;
+    private Student[] array;
+
+
+    {
```

Comment:
Is not good idea to use static block here

Reply:
fixed!